src/js/tabs/receiver.js

```diff
@@ -328,6 +328,95 @@ TABS.receiver.initialize = function (callback) {
+            rc_smoothing_protocol_e.change(function () {
+                RX_CONFIG.rcSmoothingType = $(this).val();
+                updateInterpolationView();
+            });
+            rc_smoothing_protocol_e.val(RX_CONFIG.rcSmoothingType);
+
+            var rcSmoothingnNumberElement = $('input[name="rcSmoothingInputHz-number"]');
+            var rcSmoothingnDerivativeNumberElement = $('input[name="rcSmoothingDerivativeCutoff-number"]');
+            
+            $('.tab-receiver .rcSmoothing-input-cutoff').show();
+            $('select[name="rcSmoothing-input-manual-select"]').val("1");
+            $('.tab-receiver .rc-smoothing-input-blank').hide();
+            if (RX_CONFIG.rcSmoothingInputCutoff == 0) {
+                $('.tab-receiver .rcSmoothing-input-cutoff').hide();
+                $('select[name="rcSmoothing-input-manual-select"]').val("0");
+                $('.tab-receiver .rc-smoothing-input-blank').show();
+            }
+            $('select[name="rcSmoothing-input-manual-select"]').change(function () {
+                if ($(this).val() == 0) {
+                    RX_CONFIG.rcSmoothingInputCutoff = 0;
+                    $('.tab-receiver .rcSmoothing-input-cutoff').hide();
+                }
+                if ($(this).val() == 1) {
+                    rcSmoothingnNumberElement.val(RX_CONFIG.rcSmoothingInputCutoff);
+                    $('.tab-receiver .rcSmoothing-input-cutoff').show();
+                } 
+            });
+
+            $('.tab-receiver .rcSmoothing-derivative-cutoff').show();
+            $('select[name="rcSmoothing-input-derivative-select"]').val("1");
+            $('.tab-receiver .rc-smoothing-derivative-blank').hide();
+            if (RX_CONFIG.rcSmoothingDerivativeCutoff == 0) {
+                $('select[name="rcSmoothing-input-derivative-select"]').val("0");
+                $('.tab-receiver .rcSmoothing-derivative-cutoff').hide();
+                $('.tab-receiver .rc-smoothing-derivative-blank').show();
+            }
+            $('select[name="rcSmoothing-input-derivative-select"]').change(function () {
+                if ($(this).val() == 0) {
+                    $('.tab-receiver .rcSmoothing-derivative-cutoff').hide();
+                    RX_CONFIG.rcSmoothingDerivativeCutoff = 0;
+                }
+                if ($(this).val() == 1) {
+                    $('.tab-receiver .rcSmoothing-derivative-cutoff').show();
+                    rcSmoothingnDerivativeNumberElement.val(RX_CONFIG.rcSmoothingDerivativeCutoff);
+                }
+            });
+            
+            rcSmoothingnNumberElement.change(function () {
+                RX_CONFIG.rcSmoothingInputCutoff = $(this).val();
+            });
+            rcSmoothingnNumberElement.val(RX_CONFIG.rcSmoothingInputCutoff);
+
+            rcSmoothingnDerivativeNumberElement.change(function () {
+                RX_CONFIG.rcSmoothingDerivativeCutoff = $(this).val();
```

Comment:
Just a little remark on this pattern of using change events to update output values (especially on fields that allow direct user input) - this can lead to problems, as it is possible to get a race condition between application of the limits of the input field and saving of the value, as it has surfaced in #1910.